lib/topics/state_and_behaviour.rb

```diff
@@ -1,5 +1,27 @@
 # frozen_string_literal: true
 
-# exercise state and behaviour
 module StateAndBehaviour
+  # describe cars
+  class Car
+    attr_accessor(:year, :color, :model, :current_speed)
+
+    def initialize(characteristics)
+      @year = characteristics[:year] || '1970'
```

Comment:
at the class top:
`DEFAULT_YEAR = '1970'`
and fetch:
`@year = car_options.fetch(:year, DEFAULT_YEAR) # avoid magic values`